db/organizations/kakao.eno

```diff
@@ -0,0 +1,9 @@
+name: Kakao
+website_url: https://www.kakaocorp.com/page/
+privacy_policy_url: https://www.kakao.com/policy/privacy?lang=en
+privacy_contact:
```

Comment:
I do not see an email in their privacy policy, but could this be the contact form (I did not see an English version though)? 

https://cs.kakao.com/requests?category=262&locale=ko&node=66240&service=87&select=combo2

Reply:
I don't think they have an English desk in privacy subject. Instead, the general query URL in English should be https://cs.kakao.com/requests?category=1&locale=en&node=30288&service=8&select=combo2 . If you prefer Korean version, your link is correct.

Reply:
Indeed, your link is better. Links to an English web form are best (if there is no email).